hyp3_metadata/util.py

```diff
@@ -1,28 +1,16 @@
 import re
-import shutil
 from base64 import b64encode
-from datetime import datetime
-from glob import glob
 from io import BytesIO
 from pathlib import Path
-from typing import Dict, Optional, Tuple
+from typing import Dict, Tuple
 
 from PIL import Image
 from jinja2 import Environment, PackageLoader, StrictUndefined, select_autoescape
 from osgeo import osr
 
-import hyp3_metadata
-from hyp3_metadata import data
-
 SUPPORTED_DEMS = ['EU_DEM_V11', 'GIMP', 'IFSAR', 'NED13', 'NED1', 'NED2', 'REMA', 'SRTMGL1', 'SRTMGL3', 'GLO-30']
```

Comment:
This should move into RTC if we're moving `get_dem_template_id` there

In that case, maybe both should just be class methods of the product writer